rxlifecycle-android-lifecycle/src/main/java/com/trellow/relifecycle2/android/lifecycle/RxLifecycleAndroidLifecycle.java

```diff
@@ -0,0 +1,45 @@
+package com.trellow.relifecycle2;
+
+import android.arch.lifecycle.Lifecycle;
+import android.support.annotation.NonNull;
+
+import com.trello.rxlifecycle2.LifecycleTransformer;
+import com.trello.rxlifecycle2.OutsideLifecycleException;
+
+import io.reactivex.Observable;
+import io.reactivex.functions.Function;
+
+import static com.trello.rxlifecycle2.RxLifecycle.bind;
+
+public class RxLifecycleAndroidLifecycle {
```

Comment:
Yeah, it's a bit rough here isn't it...

Minor: I'd make this class `final`. (I'm noticing this mistake in some classes I wrote before, too, will have to go correct that later.)